nexus/src/app/sagas/region_replacement_start.rs

```diff
@@ -799,8 +801,8 @@ pub(crate) mod test {
     async fn test_region_replacement_start_saga(
         cptestctx: &ControlPlaneTestContext,
     ) {
-        let mut disk_test = DiskTest::new(cptestctx).await;
-        disk_test.add_zpool_with_dataset(cptestctx.first_sled_id()).await;
```

Comment:
This isn't *strictly* necessary, but IMO it's a little clearer -- we're calling "add_zpool_with_dataset" to make a **fourth** zpool, on top of the three which already exist.